static/js/popovers.js

```diff
@@ -177,6 +177,17 @@ function render_user_info_popover(
     let can_set_away = false;
     let can_revoke_away = false;
 
+    const status_info_values = {
+        active: {
+            value: true,
```

Comment:
true and false are not very descriptive values. `"active"` and `"inactive"` would be better.